docs/examples/language_model/get_ptb_data.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 curl -O http://www.fit.vutbr.cz/~imikolov/rnnlm/simple-examples.tgz
-tar --strip-components=3 -xzvf simple-examples.tgz simple-examples/data/ptb.train.txt simple-examples/data/ptb.valid.txt simple-examples/data/ptb.test.txt
+tar --strip-components=3 -xzvf simple-examples.tgz ./simple-examples/data/ptb.train.txt ./simple-examples/data/ptb.valid.txt ./simple-examples/data/ptb.test.txt
```

Comment:
these ./ shouldn't be needed as they refer to the file inside the tarball

Reply:
I tested, only with ``./`` works on linux.